_includes/manuals/2.4/3.6_upgrade.md

```diff
@@ -31,113 +52,136 @@ For more information about how to backup your instance head over to
 
-    yum install centos-release-scl-rh
+{% highlight bash %}
+yum upgrade centos-release-scl-rh
+{% endhighlight %}
 
-or:
-
-    yum upgrade centos-release-scl-rh
-
-Clean up the yum metadata cache once more, to ensure the new repositories are used:
+Clean up the yum metadata cache:
 
-    yum clean metadata
+{% highlight bash %}
+yum clean metadata
+{% endhighlight %}
 
 Next upgrade all Foreman packages:
 
-    yum upgrade tfm\* ruby\* foreman\*
+{% highlight bash %}
+yum upgrade tfm\* ruby\* foreman\*
+{% endhighlight %}
 
+Optionally, consider removing unused SCL packages:
 
-In order to catch all configuration changes and manage them properly you should install and run
-rpmconf from the EPEL repository along with vim-enhanced (for vimdiff):
-
-    rpmconf -a --frontend=vimdiff
-
-You can skip to Step 3.
+{% highlight bash %}
+yum erase rh-ruby22\* rh-ruby24\* rh-ror42\* tfm-ror52\*
+{% endhighlight %}
+</div>
 
-##### Step 2 (B) - Debian or Ubuntu package (deb) and installer setups
+<div class="upgrade_os upgrade_os_el8">
+To upgrade an existing Foreman installation, first update with the
+appropriate foreman-release package:
 
-Upgrading from the last release to {{page.version}} has been tested on both
-Debian and Ubuntu. Updating the packages will upgrade the application and
-automatically migrate the database.
+{% highlight bash %}
+dnf upgrade https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
+{% endhighlight %}
 
-First edit `/etc/apt/sources.list.d/foreman.list` and change the release
-number from the previous release or "stable" to `{{ page.version }}`:
+Clean up the yum metadata cache:
 
-    deb http://deb.theforeman.org/ buster {{ page.version }}
-    deb http://deb.theforeman.org/ plugins {{ page.version }}
+{% highlight bash %}
+dnf clean metadata
+{% endhighlight %}
 
 Next upgrade all Foreman packages:
 
-    apt-get update
-    apt-get --only-upgrade install ruby\* foreman\*
-
-You can skip to Step 3.
-
-##### Step 2 (C) - Downloaded release (tar.bz2)
-
-- Uncompress the new program archive in a *new directory*.
-- Copy your database settings-file `config/database.yml` into the new `config` directory.
-
-VERY IMPORTANT: do NOT overwrite `config/settings.yml` with the old one.
-
-##### Step 2 (D) - git checkouts only
+{% highlight bash %}
+dnf upgrade ruby\* foreman\*
+{% endhighlight %}
 
-*Please note* that there can be significant differences between stable and
-development branches.  You MUST take care to select a branch and make sure you
-get the correct one.
+Foreman runs best on PostgreSQL 12 but the default version on EL8 is 10. The installer will try to use version 12 but can't upgrade existing databases. An upgrade can be performed by [switching module streams](https://docs.fedoraproject.org/en-US/modularity/using-modules-switching-streams/).
 
-1. Go to the Foreman root directory and run the following command:
+First make sure you have version 10 installed:
```

Comment:
The wording here implies I *should* have version 10 installed, I think you may want this to be more like "First check to see what version of postgresql-server is currently installed"